server/app/assets/javascripts/sorting.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * ElementSorter provides utilities for sorting a group of objects.
+ */
+class ElementSorter {
+  private static sortableElementSelector = '.cf-sortable-element'
+
+  static sortElementsOnLoad({
+    parent,
+    compareFunc,
+  }: {
+    parent: HTMLElement
+    compareFunc: (first: Element, second: Element) => number
+  }) {
+    const elements = Array.from(
+      parent.querySelectorAll(ElementSorter.sortableElementSelector),
+    )
+    try {
+      ElementSorter.sortCards(parent, elements, compareFunc)
+    } finally {
+      // Make sure to always show the element, even
+      // if there was an error while sorting.
+      parent.classList.remove('invisible')
```

Comment:
That makes me a bit nervous that we rely on JS to make element visible. So if JS is slow to load user will see empty page. I think ideally we'd want to sort server-side and add ability to re-sort client-side if necessary. But it's more work to maintain both server-side and client-side sorting... Do we expect other client-side sorting options to be added soon?